### ISSUE TITLE
White line persists after removing the Open-Shell

### ISSUE THREAD

**Issue by ironman199999** [Open-Shell/Open-Shell-Menu#2007]
### Describe the bug

![Screenshot 2024-10-12 160939](https://github.com/user-attachments/assets/23adbfc8-e699-49e9-bfef-d23795146838)
even after using utility and removing all the files, the white line could not be removed

### Area of issue

Windows Explorer

### To reproduce

1. Install Open-Shell
2. Uninstall using Utility.exe
3. Remove any files .
4. open File Explorer

### Expected behavior

The Explorer should have returned to default

### Open-Shell version

4.4.191

### Windows version

Windows 11 24H2

**Comment by Merigold117** [comment]
I'm having the same issue, going on regedit and locking the toolbar also didn't seem to work

**Comment by facespkz** [comment]
I solved it by disabling the Internet Explorer toolbar: https://github.com/Open-Shell/Open-Shell-Menu/issues/1513#issuecomment-1488015980

**Comment by Merigold117** [comment]
Hey everyone, I was able to fully remove the white bar from showing up in my file explorer by doing the following:

1. Reinstall Open-Shell Menu along with Classic File Explorer.
2. Download Utility.exe from the [Releases](https://github.com/Open-Shell/Open-Shell-Menu/releases) page.
3. Use Utility.exe to fully uninstall Open-Shell Menu and Classic File Explorer.

After that I no longer see the white bar appearing on my file explorer ever again.

**Comment by ge0rdi** [comment]
@Merigold117 
Just to clarify, you specifically had to uninstall OS using `Utility`, right?
Doing _standard_ uninstall didn't help? (have you tried that?)

**Comment by Merigold117** [comment]
> @Merigold117 Just to clarify, you specifically had to uninstall OS using `Utility`, right? Doing _standard_ uninstall didn't help? (have you tried that?)

Yup, using the standard uninstall method is what caused the white bar on file explorer to show up for. Using Utility.exe is what fixed my problem.

**Comment by xe3a** [comment]
The issue is still actual. how to return the original line of the Explorer without full deinstalling Open Shell?

**Comment by lilliancf** [comment]
I still have this issue even after full removal of open shell

**Comment by Turk303** [comment]
not for everyone but to get the desired look maybe try a different theme & ultrauxthemepatcher

Here's a fix on the net I found that worked for me.

im happy with mine looking like this

<a href="https://ibb.co/ycn9Ghjp"><img src="https://i.ibb.co/B5Vpd4MG/2025-03-10-06-03-55-Classic-Explorer-Bar-Issue-2.png" alt="2025-03-10-06-03-55-Classic-Explorer-Bar-Issue-2" border="0"></a>
